source/js/interview.js

```diff
@@ -185,9 +185,9 @@ $(document).ready(function () {
 
   // Function that's responsible for generating a hash from a string
```

Comment:
(I might have asked in previous PR) what kind of hash is this? for what purpose? there are many different kinds of hashes

Reply:
to generate a hash code that can be used to identify certain things.
the sole purpose and function of this function is to just generate a hash from given string